ClinGen/clincoded
# Connects to #947. Connects to #937. Connects to #978. Disease-dependency overhaul. Fix rendering on Windows.

#### Connects to #947:

* Disabled disease-dependent logic for rendering and saving data in `form.js`
* Updated form rendering to support HTML codes form json for descriptions
* Fixed typos in descriptions

Testing:

1. Load VCI, begin interpretation
2. Confirm that the correct criteria are marked as 'Disease-specific'
3. Confirm that disease-specific criteria are still available for editing and saving with no disease being associated to interpretation
3. Save interpretation with and without disease-specific tags, with and without disease associated to interpretation, and confirm that things work

#### Connects to #937:

Testing:

1. Load VCI and begin interpretation on Windows machine (Chrome, since FF login is broken)
2. Check PP3 (missense subtab) and PVS1 (lof subtab) and confirm that the text in the description box does not ellipsis.

![image](https://cloud.githubusercontent.com/assets/4326866/18066113/cde3b2fa-6deb-11e6-8bfe-fae6e0f43f14.png)
![image](https://cloud.githubusercontent.com/assets/4326866/18066137/dffa977e-6deb-11e6-8223-ce1c7a81189c.png)

#### Connects to #978:

Testing:

1. Load VCI and begin interpretation
2. Confirm that the 'Disease dependent' labels on the form description boxes now say 'Disease-specific'

![image](https://cloud.githubusercontent.com/assets/4326866/18291063/ee0e9130-743a-11e6-8254-2f5e261559ac.png)
